src/views/MyRecords.jsx

```diff
@@ -13,10 +13,10 @@ export class MyRecords extends Component {
           <Headimg ImgUrl={imgHeadMyRecords} Text={`Mis Registros`} />
         </View>
         <ScrollView style={MyRecordStyle.scrollView}>
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}} />
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
         </ScrollView>
       </View>
```

Comment:
Next time, try to use [FlatList](https://reactnative.dev/docs/flatlist), I know that it's just a preview, but you can pass an array with any elements 😶‍🌫️

Reply:
thank you for the suggestion !!